files/provisioning_job.yml

```diff
@@ -66,44 +66,62 @@
                          default: ''
                          description: 'The arguments to be passed to the kernel'
                  - string:
-                         name: kernel_path
-                         default: ''
-                         description: 'The path to the kernel file to be (non compulsory) uploaded'
-                 - string:
-                         name: initrd_path
-                         default: ''
-                         description: 'The path to the initrd file to be (non compulsory) uploaded'
+                         name: client_branch
+                         default: 'master'
+                         description: 'The branch of the mrp client to use'
                  - string:
-                         name: preseed_path
-                         default: ''
-                         description: 'The path to the preseed file to be (non compulsory) uploaded'
-                 - choice:
-                         name: branch
-                         choices:
-                            - master
-                            - bger
-                            - rengolin
+                         name: automation_branch
                          default: 'master'
-                         description: 'The branch of the provisioning job to use'
+                         description: 'The Ansible logic branch to use'
          builders:
                  - shell: |
-                         cd ${WORKSPACE}
+                         #!/bin/bash
                          mname=$( echo $machine_name| cut -d',' -f 1)
+                         mrp_url=http://10.40.0.11:5000
```

Comment:
I wish this was a config somewhere, but that's for another patch.

Reply:
Yup....